invenio_app_ils/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -8,16 +8,34 @@
           "type": "keyword"
         },
         "title": {
-          "type": "keyword"
+          "properties": {
+            "title": {
+              "type": "keyword"
```

Comment:
I think for all these text fields we should have `type: text` instead.
Shall we review together the mappings if you like?